### PR TITLE
fix(readme):  Updated readme

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,141 +1,90 @@
-# PolkaStats
+# Cerestats
 
-Polkastats is a block explorer for [Polkadot](https://polkadot.network/), [Kusama](https://kusama.network/) and other blockchains based on [Substrate](https://substrate.dev/) framework.
```
Comment (AndreiNavoichyk) [inline README.md:3]: We need to have a link to Cerestats PROD

File: README.md [hunk 2]
```diff
@@ -1,141 +1,90 @@
...
 ## Install
 
 Install mono-repo:
 
-```
-cd /usr/local/
-git clone https://github.com/Colm3na/polkastats-ng.git
-cd polkastats-ng
+```bash
+git clone git@github.com:Cerebellum-Network/cerestats.git
```
Comment (AndreiNavoichyk) [inline README.md]: You should use `https` scheme

File: README.md [hunk 3]
```diff
@@ -1,141 +1,90 @@
...
+cd cerestats
 yarn
 ```
 
 ### Backend
 
-Before build dockers be sure to edit `HASURA_GRAPHQL_ADMIN_SECRET` environmment variable in the proper docker-compose file `backend/docker/docker-compose-polkadot.yml` or `backend/docker/docker-compose-kusama.yml`.
+Before build dockers be sure to edit `HASURA_GRAPHQL_ADMIN_SECRET` environmment variable in the proper docker-compose file `backend/docker/docker-compose-cere-mainnet.yml`.
 
 Polkadot:
```
Comment (AndreiNavoichyk) [inline README.md]: Cere Network

File: README.md (hunk 1 above)
Comment (karthikeyan-cere) [reply README.md:3]: Added

File: README.md (hunk 2 above)
Comment (karthikeyan-cere) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (karthikeyan-cere) [reply README.md]: Updated

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**